Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Compositor/NativeInterface.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView.Compositor
+{
+    public static class NativeInterface
```

Comment:
>NativeInterface [](start = 24, length = 15)

Does it feel worth having this class only contain functionality for UNITY_EDITOR compared to always having these dllimports even when the dll's wont exist?